RoadMap_SwiftUI/RoadMap_SwiftUI/PurchaseView.swift

```diff
@@ -0,0 +1,133 @@
+//
+//  PurchaseView.swift
+//  RoadMap_SwiftUI
+//
+//  Created by Артур Щукин on 20.01.2023.
+//
+
+import SwiftUI
+
+/// Экран выбора тарифа
+struct PurchaseView: View {
+    
+    // MARK: - Public properties
+    var body: some View {
+        NavigationView {
+            ScrollView {
+                ZStack {
+                    VStack {
+                        companyTaglineTextView
+                        vpnPlusOptionsStackView
+                    }
+                }
+            }
+        }
+    }
+    
+    // MARK: - Private properties
+    @EnvironmentObject private var viewModel: VPNViewModel
+    @State private var isBuyVactionButtonTapped = false
+    @State private var isStandardVactionButtonTapped = false
+    
+    private var vpnPlusOptions = [
+        VpnPlusOption(option: Constants.Texts.vacationText, price: Constants.Numbers.vacationPrice),
+        VpnPlusOption(option: Constants.Texts.standardText, price: Constants.Numbers.standardPrice)
+    ]
+    
+    private var companyTaglineTextView: some View {
+        VStack(spacing: Constants.Numbers.verticalSpacing) {
+            Text(Constants.Texts.companyTagline)
+                .fontWeight(.bold)
+                .font(.largeTitle)
+                .foregroundColor(.white)
+            Text(Constants.Texts.protectMessageText)
+                .font(.headline)
+                .foregroundColor(.white)
+        }
+    }
+    
+    private var vpnPlusOptionsStackView: some View {
+        VStack(spacing: Constants.Numbers.vpnPlusVerticalSpacing) {
+            vacationCalendarImageView
+            vacationOptionTextView
+            buyVacationOptionButtonView
+            standardCalendarImageView
+            standardOptionTextView
+            buyStandardOptionButtonView
+        }
+    }
+    
+    private var vacationCalendarImageView: some View {
+        fullCalendarImage(day: Constants.Texts.seven)
+    }
+    
+    private var vacationOptionTextView: some View {
+        HStack {
+            Text(vpnPlusOptions[0].option)
```

Comment:
```suggestion
небезопасно, добавить проверку
            Text(vpnPlusOptions[0].option)
```
а раз это уже логика, то вынести во вьюмодель